data/Templates/Stu3ToR4/DataType/Dosage/Dosage.liquid

```diff
@@ -0,0 +1,16 @@
+{% mergeDiff msg -%}
```

Comment:
Do we need to make this file starting with `_`, as it is a snippet instead of root template?